frontend/src/components/orders/OfferItem.vue

```diff
@@ -0,0 +1,93 @@
+<script setup lang="ts">
+import Button from "primevue/button";
+import Card from "primevue/card";
+import Dialog from "primevue/dialog";
+import FloatLabel from "primevue/floatlabel";
+import InputNumber from "primevue/inputnumber";
+import InputText from "primevue/inputtext";
+import Textarea from "primevue/textarea";
+import { ref } from "vue";
+
+import { getArticles } from "@/backendClient";
+import { ArticleKind } from "@/global/types/appTypes";
+import type { Article } from "@/global/types/entities";
+
+defineProps<{
+  id: string;
+  type: ArticleKind;
+}>();
+
+let isArticlesListVisible = ref(false);
+const articlesList = ref<Article[]>([]);
+
+async function openArticlesList() {
+  isArticlesListVisible.value = true;
+  articlesList.value = (await getArticles()).data;
+}
+
+let offerInfo = ref({
+  title: "",
+  description: "",
+  number: 0,
+  unit: "",
+  price: 0,
```

Comment:
Should these values be null if the ArticleKind is a heading?

Reply:
I don't use these fields in the heading.

Reply:
True. But if we even set them to these values it is likely that we will forward this to the backend.

if those fields are not applicable, we shouldn't set them to keep the data clean.